packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -124,9 +124,9 @@ export class AtomicCategoryFacet
   @Prop() public withSearch = false;
   /**
    * The sort criterion to apply to the returned facet values.
-   * Possible values are 'score', 'alphanumeric', 'occurrences', and 'automatic'.
-   * TODO: add automatic (occurences when not expanded, alphanumeric when expanded)
+   * Possible values are 'alphanumeric' and 'occurrences'.
    */
+  // TODO: add automatic (occurrences when not expanded, alphanumeric when expanded)
```

Comment:
I _think_ this will omit the TODO from getting into the docs.

Reply:
You can verify that pretty easily by building the project `npm run build` in the atomic package, and then opening `docs/atomic.docs.json` or `docs/atomic.docs.d.ts`

Reply:
Yep, this removed the TODO from the parsed output.